Remove link and add logging levels (#6484)
OpenLiberty/docs: modules/ROOT/pages/jdbc-tracing.adoc

- `oracleLogFormat`: SimpleFormatter (the simple formatter from java.util.logging)
- `oracleLogTraceLevel`: INFO

> For Oracle recommended settings, see their support page.

Reviewer: ```suggestion
For Oracle recommended settings, see the Oracle documentation.
```
